### PR TITLE
Convert to markdown, formatting, proofreading, enhancing.

### PR DESCRIPTION
**Pull request by GBLodb**
title.

### CODE REVIEW

**Comment by GBLodb** [comment]
> Hi @GBLodb, could you please separate the single commit in to `2 commits` just like this:
> 
>     1. "mv to .md for some file" (using `git mv` commend to create a rename commit)
> 
>     2. "convert to markdown, etc" (used to markdown convertion and sync with english version)
> 
> 
> You can see [this pr](https://github.com/mdn/translated-content/pull/5799/commits) for example. Thanks.
> 
> If it's a bit difficult to keep working on this pr, new pr is welcomed.

Working on it.

**Comment by yin1999** [comment]
As a suggestion, it's better to follow **one pr with a single job**, if you mainly deal with markdown convertion, just create a new pr to deal with this file: `files/zh-cn/learn/html/index.md`.

**Comment by GBLodb** [comment]
@yin1999 

Sorry for this mess in the conversation, it should be ready for review now.

By the way, do zh-CN translators share a communication group like zh-TW? A QQ group or something would be great.

**Review by yin1999 — CHANGES_REQUESTED** [review]
请先根据这两处问题修改文档

**Comment by yin1999** [comment]
> By the way, do zh-CN translators share a communication group like zh-TW? A QQ group or something would be great.

Sorry about that, I'm not going to do that now (It may take a lot of effort to manage).

File: files/zh-cn/learn/front-end_web_developer/index.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
+---
+title: Front-end web developer
```
Comment (yin1999) [inline files/zh-cn/learn/front-end_web_developer/index.md:2]:
```suggestion
title: Web 前端开发者
```

File: files/zh-cn/learn/front-end_web_developer/index.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+#### 我怎么知道自己是否可以继续往下学习了?
+
+每个模块的评估练习都是针对测试你主题相关知识掌握能力设计的，完成了每个练习就表示你可以继续往后学习下一个模块了。
+
+#### 核心模块
+
+- [学习 CSS 的第一步](/zh-CN/docs/Learn/CSS/First_steps) (10–15 小时阅读/练习)
+- [编写 CSS](/zh-CN/docs/Learn/CSS/Building_blocks) (35–45 小时阅读/练习)
+- [添加文本样式](/zh-CN/docs/Learn/CSS/%E4%B8%BA%E6%96%87%E6%9C%AC%E6%B7%BB%E5%8A%A0%E6%A0%B7%E5%BC%8F) (15–20 小时阅读/练习)
```
Comment (yin1999) [inline files/zh-cn/learn/front-end_web_developer/index.md:102]: 这处的链接指向原来的文档路径，需要更新：

```suggestion
- [添加文本样式](/zh-CN/docs/Learn/CSS/styling_text) (15–20 小时阅读/练习)
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/learn/front-end_web_developer/index.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+仅需基础的计算机知识。
+
+#### 我怎么知道自己是否可以继续往下学习了?
+
+本章节没有评估测试，但是请不要跳过它 — 这对于帮你准备好应对后面的练习和学习非常重要。
+
+#### 核心提示
+
+- [安装基础软件](/zh-CN/docs/Learn/Getting_started_with_the_we-b/Installing_basic_software)  (15 分钟)
```
Comment (yin1999) [inline files/zh-cn/learn/front-end_web_developer/index.md:64]:
```suggestion
- [安装基础软件](/zh-CN/docs/Learn/Getting_started_with_the_web/Installing_basic_software)  (15 分钟)
```

File: files/zh-cn/learn/front-end_web_developer/index.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+每个模块的评估练习都是针对测试你主题相关知识掌握能力设计的，完成了每个练习就表示你可以继续往后学习下一个模块了。
+
+#### 核心模块
+
+- [Web 表单](/zh-CN/docs/Learn/HTML/Forms) (40–50 小时)
+
+### 让所有人都能使用 Web
+
+学习用时: 60–75 小时
```
Comment (yin1999) [inline files/zh-cn/learn/front-end_web_developer/index.md:147]:
```suggestion
学习用时: 45-55 小时
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
请完成这两处更高，并使用 `git rebase -i c6451a60499dd6f69689cb5cd7b0f29c5dd8414b` 命令，将最后的这几次 commit 合并到 convertion commit。参考：<https://backlog.com/git-tutorial/cn/stepup/stepup7_5.html>

**Pull request merged**